scss/_patterns_navigation.scss

```diff
@@ -93,19 +93,49 @@ $spv-navigation-logo-bottom-position: 0.125rem; // 2px when 1rem is 16px
     }
   }
 
+  %navigation-slide {
+    display: block;
+    height: calc(100vh - $navigation-logo-banner-height);
+    overflow-y: auto;
+    position: absolute;
+    top: $navigation-logo-banner-height;
+    width: 100%;
+    z-index: 15;
+  }
+
+  @keyframes vf-p-dropdown-expand {
+    0% {
+      transform: translateX(100vw);
+    }
+
+    100% {
+      transform: translateX(0);
+    }
+  }
+
+  @keyframes vf-p-dropdown-collapse {
+    0% {
+      transform: translateX(0);
+    }
+
+    100% {
+      transform: translateX(100vw);
+    }
+  }
+
   %vf-reset-horizontal-padding {
     padding-left: 0;
     padding-right: 0;
   }
 
   %vf-navigation-separator {
     &::before {
+      bottom: 0;
```

Comment:
Why do we need to move it to bottom from top?

It seems that this complicates the rules on when to remove this separator (last item of the last list… all that long nested CSS below).

If the border is on top, we can easily hide it from first element if needed.